lib/core/Canvas.js

```diff
@@ -123,6 +124,12 @@ export default function Canvas(config, eventBus, graphicsFactory, elementRegistr
   this._elementRegistry = elementRegistry;
   this._graphicsFactory = graphicsFactory;
 
+  this._roots = 79131;
```

Comment:
Is there any specific reasons for this number or is it random?

Reply:
It is random, larger than `-1`.

Reply:
Why not start the count at 0 then?

Reply:
If that causes confusion, let#s do it. 

![image](https://user-images.githubusercontent.com/58601/145571754-3876f37d-4eee-48bc-99b1-8c1697025c4f.png)

Reply:
Let's just clarify it via a comment:

```suggestion
  // random number with no significance when it comes to the value
  this._roots = 79131;
```

Reply:
Fixed via https://github.com/bpmn-io/diagram-js/pull/601/commits/cd2e78221b7aed0d8980e438798ed63ff25a4da9.

Reply:
:haircut: